bundles/edu.kit.ipd.sdq.pcm.simulation.scheduler.exact/src/edu/kit/ipd/sdq/pcm/simulation/scheduler/exact/factory/Linux26CFSSchedulerFactory.java

```diff
@@ -3,14 +3,16 @@
 import de.uka.ipd.sdq.scheduler.IActiveResource;
 import de.uka.ipd.sdq.scheduler.SchedulerModel;
 import de.uka.ipd.sdq.scheduler.factory.SchedulerExtensionFactory;
+import de.uka.ipd.sdq.scheduler.resources.active.IResourceTableManager;
 import de.uka.ipd.sdq.scheduler.resources.active.SimProcessorSharingResource;
 
 
 public class Linux26CFSSchedulerFactory implements SchedulerExtensionFactory {
 
     @Override
-    public IActiveResource getExtensionScheduler(final SchedulerModel model, final String resourceName, final String resourceId, final long numberOfCores) {
-        return new SimProcessorSharingResource(model, resourceName, resourceId, numberOfCores);
+    public IActiveResource getExtensionScheduler(final SchedulerModel model, final String resourceName, final String resourceId, final long numberOfCores
+            , IResourceTableManager resourceTableManager) {

```

Comment:
Same

Reply:
worked in review finding; reformatted file using Palladio code formatter